### ISSUE TITLE
Seems not working on Notepad++ v7.8.1 (x64)

### ISSUE THREAD

**Issue by alleusai** [dail8859/LuaScript#79]
Hi, after put LuaScript.dll in LuaScript folder inside the plugins folder the message LuaScript.dll is not compatible with the current version of Notepad++. Specified module not found.

Regards,
Alessandro

**Comment by alleusai** [comment]
The OS is Win10 64bit v.1803 build 17134.1130.
Regards,
Alessandro

**Comment by xylographe** [comment]
You cannot combine any version of LuaScript with any version of NPP. Newer versions of LuaScript expect that certain functionality from NPP/Scintilla is available. Hence, to be safe, use the NPP installer (it will also install the list of appropriate plug-ins), then use PluginAdmin to install the appropriate LuaScript plug-in.

To use the most recent version of LuaScript (manual install), update to NPP v7.8.3. Make sure that you replace _both_ assemblies,
```
\plugins\LuaScript\LuaScript.dll
\plugins\LuaScript\lua53.dll
```

**Comment by alleusai** [comment]
Hi xylographe.
I did following steps:
- NPP v7.8.3 install
- try to install LuaScript from PluginAdmin (v 0.8 is available) with no luck: folder LuaScript isn't in NPP's plugins folder
- download from https://github.com/dail8859/LuaScript/releases v0.10 version (LuaScript_v0.10_x64.zip): zip file contains only LuaScript.dll
- create LuaScript folder in NPP's plugins folder
- copy LuaScript.dll from zip file to LuaScript folder
- run NPP: message LuaScript.dll is not compatible with the current version of Notepad++. Specified module not found.

Hope this will help.
Thanks,
Alessandro

**Comment by dail8859** [comment]
@alleusai Yes that level of detail helps immensely.

Looks like the appveyor script isn't packaging the Lua DLL along with the plugin.

**Comment by dail8859** [comment]
Closed by 8d479234c7cd2c7de2e2de67ab11a49bdf577ab0

While you wait for this to show up in the Plugin Admin, you can manually download the new release here:
https://github.com/dail8859/LuaScript/releases/tag/v0.10.1
